examples/tabular/40_advanced/example_custom_configuration_space.py

```diff
@@ -54,6 +54,15 @@ def get_search_space_updates():
                    hyperparameter='ResNetBackbone:dropout',
                    value_range=[0, 0.5],
                    default_value=0.2)
+    updates.append(node_name='network_backbone',
+                   hyperparameter='ResNetBackbone:multi_branch_choice',
+                   value_range=['shake-shake'],
+                   default_value='shake-shake')
+    updates.append(node_name='network_backbone',
```

Comment:
Is it ok to have the same `node_name`?

Reply:
what do you mean?

Reply:
I guess he means having `network_backbone` for both updates as node names although the hyperparameter name changes.

Reply:
yeah, that's totally fine. Do you see a problem with it @nabenabe0928 ?